app/src/main/java/org/mozilla/fenix/session/SessionNotificationService.kt

```diff
@@ -50,13 +50,23 @@ class SessionNotificationService : Service() {
                 metrics.track(Event.PrivateBrowsingNotificationTapped)
                 components.core.sessionManager.removeAndCloseAllPrivateSessions()
 
-                if (!VisibilityLifecycleCallback.finishAndRemoveTaskIfInBackground(this)) {
-                    startActivity(
-                        Intent(this, HomeActivity::class.java).apply {
-                            this.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
-                        }
-                    )
+                val homeScreenIntent = Intent(this, HomeActivity::class.java);
+                val intentFlags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
+                if (VisibilityLifecycleCallback.finishAndRemoveTaskIfInBackground(this)) {
+                    // Set browsing mode to 'regular' and it's startmode to be in background (recents screen)
+                    homeScreenIntent.apply {
+                        setFlags(intentFlags)
+                        putExtra(HomeActivity.PRIVATE_BROWSING_MODE, false)
```

Comment:
There's still some discussion if it should be `true` or `false` for the private browsing, I'll keep you posted

Reply:
It should be false for the moment